plugins/scaffolder-node/src/tasks/types.ts

```diff
@@ -37,7 +37,8 @@ export type TaskStatus =
   | 'completed'
   | 'failed'
   | 'open'
-  | 'processing';
+  | 'processing'
```

Comment:
I feel like the types in this file could also be relocated `@backstage/plugin-scaffolder-common`..? For example, `TaskStatus` is pretty much the same as `ScaffolderTaskStatus`.